PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -371,6 +397,7 @@ def nanoAOD_customizeCommon(process):
                                      addParticleNet=nanoAOD_addDeepInfoAK8_switch.nanoAOD_addParticleNet_switch,
                                      jecPayload=nanoAOD_addDeepInfoAK8_switch.jecPayload)
     (run2_nanoAOD_94XMiniAODv1 | run2_nanoAOD_94X2016 | run2_nanoAOD_94XMiniAODv2 | run2_nanoAOD_102Xv1 | run2_nanoAOD_106Xv1).toModify(process, lambda p : nanoAOD_addTauIds(p))
+    (run2_nanoAOD_94XMiniAODv1 | run2_nanoAOD_94X2016 | run2_nanoAOD_94XMiniAODv2 | run2_nanoAOD_102Xv1 | run2_nanoAOD_106Xv1 | run2_nanoAOD_106Xv2).toModify(process, lambda p : nanoAOD_addBoostedTauIds(p))
```

Comment:
See my message in the main thread. In case of standard taus, the reproduction of IDs is not the default, but necessery for several eras, which explains the line above. In your case, producing boosted tau IDs will be the default for now and you need to deactivate it for past eras, i.e. run2_nanoAOD_106Xv1 and earlier.